integ-test/src/test/java/com/amazon/opendistroforelasticsearch/sql/sql/FlowConrolFunctionIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ *    Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License").
+ *    You may not use this file except in compliance with the License.
+ *    A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file. This file is distributed
+ *    on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *    express or implied. See the License for the specific language governing
+ *    permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.sql;
+
+import static com.amazon.opendistroforelasticsearch.sql.legacy.TestsConstants.TEST_INDEX_ACCOUNT;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.hitAny;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.kvInt;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.kvString;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.io.IOException;
+
+
+import com.amazon.opendistroforelasticsearch.sql.legacy.SQLIntegTestCase;
+import com.amazon.opendistroforelasticsearch.sql.util.TestUtils;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.common.xcontent.LoggingDeprecationHandler;
+import org.elasticsearch.common.xcontent.NamedXContentRegistry;
+import org.elasticsearch.common.xcontent.XContentFactory;
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.search.SearchHits;
+import org.json.JSONObject;
+
+import org.junit.Assume;
+import org.junit.Test;
+
+
+/**
+ * Created by allwefantasy on 8/25/16.
+ */
+public class FlowConrolFunctionIT extends SQLIntegTestCase {
+
+  @Override
+  public void init() throws Exception {
+    super.init();
+    TestUtils.enableNewQueryEngine(client());
+    loadIndex(Index.BANK);
+    loadIndex(Index.ACCOUNT);
+    loadIndex(Index.ONLINE);
+    loadIndex(Index.DATE);
+  }
```

Comment:
It's fine to load only the account index and remove others since only the account index is used in this IT class. Could you also add some test cases over the null/missing values. The index `bank_with_null_values.json` might be helpful since it contains a couple of null or missing values in some fields.

Reply:
Got it, thanks